analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Monitor.Conditions.vb

```diff
@@ -188,6 +188,17 @@ Namespace Monitor_Conditions
             End If
         End Sub
 
+        Public Sub Method24(Condition As Boolean)
+            Dim isAcquired As Boolean = False
```

Comment:
In general, VB.NET doesn't suffer from the structural issue with casing. So all variables and parameters are typically in normal case.
`= False` is not needed - compiler knows what the default value is. Developper doesn't need to tell the language 😆 
```suggestion
            Dim IsAcquired As Boolean
```